metron-analytics/metron-profiler/src/main/flux/profiler/remote.yaml

```diff
@@ -25,11 +25,28 @@ components:
     -   id: "defaultExecutor"
         className: "org.apache.metron.profiler.stellar.DefaultStellarExecutor"
 
-    -   id: "hbaseMapper"
-        className: "org.apache.metron.profiler.bolt.ProfileHBaseMapper"
+    -   id: "rowKeyBuilder"
+        className: "org.apache.metron.profiler.hbase.SaltyRowKeyBuilder"
         properties:
             - name: "saltDivisor"
               value: ${profiler.hbase.salt.divisor}
+            - name: "periodsPerHour"
+              value: ${profiler.periods.per.hour}
+
+    -   id: "columnBuilder"
+        className: "org.apache.metron.profiler.hbase.ValueOnlyColumnBuilder"
+        constructorArgs:
+            - "${profiler.hbase.column.family}"
+
+    -   id: "hbaseMapper"
+        className: "org.apache.metron.profiler.bolt.ProfileHBaseMapper"
+        properties:
+            - name: "rowKeyBuilder"
+              ref: "rowKeyBuilder"
+            - name: "columnBuilder"
+              ref: "columnBuilder"
+            - name: "executor"
+              ref: "defaultExecutor"
```

Comment:
I have multiple components that each need their own `StellarExecutor`. Does Flux treat the `defaultExecutor` reference as a singleton and so every reference to `defaultExecutor` is pointing to the same object?  Or is treated more like a Spring Bean's 'Prototype' scope?  I would think it would have to be treated as a prototype, but I'd like to make sure.